.github/workflows/review.yml

```diff
@@ -2,14 +2,27 @@ name: review
 
 on:
   pull_request:
-    types: [opened, reopened, synchronize]
+    branches:
+      - "*"
 
 jobs:
   make:
     runs-on: ubuntu-latest
     steps:
       - name: Setup Sage
         uses: einride/sage/actions/setup@master
+        with:
+          go-version: 1.19
 
       - name: Make
         run: make
+
+      - name: Dry-run semantic-release
+        run: make semantic-release repo=${{ github.repository }} dry=true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Dry-run GoReleaser
+        run: make go-releaser snapshot=true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What issues would we catch by running this in review?

Reply:
That it can be built without error for all target architectures